dashboard/src/main/kotlin/com/ritense/dashboard/web/rest/dto/DashboardWidgetDataResultDto.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.dashboard.web.rest.dto
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped
+
+class DashboardWidgetDataResultDto(
```

Comment:
Consider making this an abstract class and having several implementations. That way we can still be more specific  with what the result looks like.

Reply:
As discussed, an abstract doesn't make much sense.